gatk-launch

```diff
@@ -92,6 +92,9 @@ def main(args):
             print("")
             print("    ./gatk-launch Tool --help  Print help on a particular tool" )
             print("")
+            print(" Configuration File Specification")
+            print("     --gatkConfigFile                PATH/TO/GATK/PROPERTIES/FILE")
```

Comment:
I'm happy to change to the POSIX syntax.  I don't want to change to argparse for this PR, but we can open a new issue for it.